src/metadataTextTracks.js

```diff
@@ -0,0 +1,123 @@
+import videojs from 'video.js';
```

Comment:
On the other hand, I wouldn't mind going even further with a name like `adCues.js`, since that fully communicates the goal, even if the implementation is technically not so specific.